dataplane/config/config.go

```diff
@@ -121,6 +121,74 @@ func configRead(c *cli.Context) error {
 	return nil
 }
 
+// CheckConfigAndCommandFlagsDP : CAAS does not require workspace check
+func CheckConfigAndCommandFlagsDP(c *cli.Context) error {
+	err := fl.CheckRequiredFlagsAndArguments(c)
+	if err == nil {
+		return configReadDP(c)
+	}
+	return err
+}
+
+func configReadDP(c *cli.Context) error {
+	args := c.Args()
```

Comment:
Contains many code duplication

Reply:
Yes there are , There are configRead function calls api calls to check workspace which is not required for caas. So thought of creating one. This can also be achieved by passing a boolean flag to configRead function which will skip workspace related checks. Do you recommend that?

Reply:
I don't recommend a boolean flag, but what i recommend is to create a `configRead` without workspace, which could be used from both places, and create a `configSetWorkspace` and call it where workspace configuration is necessary.